app/src/play/java/de/danoeh/antennapod/activity/CastEnabledActivity.java

```diff
@@ -83,15 +111,21 @@ public boolean onPrepareOptionsMenu(Menu menu) {
     @Override
     protected void onResume() {
         super.onResume();
+        if (!CastManager.isInitialized()) {
+            return;
+        }
         castButtonVisibilityManager.setResumed(true);
     }
 
     @Override
     protected void onPause() {
         super.onPause();
-        castButtonVisibilityManager.setResumed(false);
```

Comment:
Why was this line removed? Isn't the whole `onPause` function useless now?

Reply:
good catch - it was a honest mistake on my part.